.gitignore

```diff
@@ -2,6 +2,7 @@
 __pycache__
 *.pyc
 /.venv
+.eggs
```

Comment:
Cool, we have green eggs. We just need some ham now.

Reply:
:heavy_check_mark: